### PR TITLE
Update Flask tutorial

### PR DESCRIPTION
**Pull request by luabud**

### CODE REVIEW

**Comment by luabud** [comment]
@dawnwages  -- JFYI this PR will update two screenshots that are in the Django tutorial as well!

**Review by cwebster-99 — APPROVED** [review]

File: docs/python/tutorial-flask.md [hunk 1]
```diff
@@ -390,7 +358,7 @@ The following sections demonstrate both types of static files.
 
     ```html
     {%if name %}
-        <span class="message">Hello there, \{{ name }}!</span> It's \{{ date.strftime("%A, %d %B, %Y at %X") }}.
```
Comment (gregvanl) [inline docs/python/tutorial-flask.md:393]: We need to leave the backslash escape characters here.
It's a Gotcha that hurks page rendering https://github.com/microsoft/vscode-docs/blob/main/CONTRIBUTING.md#double-opening-curly-braces-break-generated-handlebar-files

File: docs/python/tutorial-flask.md [hunk 2]
```diff
@@ -434,22 +402,22 @@ A base page template in Flask contains all the shared parts of a set of pages, i
...
-1. In the `templates` folder, create a file named `layout.html` with the contents below, which contains blocks named "title" and "content". As you can see, the markup defines a simple nav bar structure with links to Home, About, and Contact pages, which you create in a later section. Each link again uses Flask's `url_for` tag to generate a link at runtime for the matching route.
+1. In the `templates` folder, create a file named `layout.html` with the contents below, which contains blocks named "title" and "content". As you can see, the markup defines a simple nav bar structure with links to Home, About, and Contact pages, which you will create in a later section. Each link again uses Flask's `url_for` tag to generate a link at runtime for the matching route.
 
     ```html
     <!DOCTYPE html>
     <html>
         <head>
             <meta charset="utf-8" />
             <title>{% block title %}{% endblock %}</title>
-            <link rel="stylesheet" type="text/css" href="\{{ url_for('static', filename='site.css')}}" />
```
Comment (gregvanl) [inline docs/python/tutorial-flask.md:445]: Same here need to leave the `\` here and below

File: docs/python/tutorial-flask.md [hunk 3]
```diff
@@ -502,11 +470,11 @@ You can run the app at this point, but because you haven't made use of the base
 
 Because the three pages you create in the next section extend `layout.html`, it saves time to create a **code snippet** to initialize a new template file with the appropriate reference to the base template. A code snippet provides a consistent piece of code from a single source, which avoids errors that can creep in when using copy-paste from existing code.
 
-1. In VS Code, select the **File** (Windows/Linux) or **Code** (macOS), menu, then select **Preferences** > **User snippets**.
+1. In VS Code, select **File** >  **Preferences** > **Configure User snippets**.
```
Comment (gregvanl) [inline docs/python/tutorial-flask.md:473]: Any reason you removed this? Mac users have **Preferences** under **Code** and they often get confused when we only mention the **File** menu

File: docs/python/tutorial-flask.md (hunk 1 above)
Comment (gregvanl) [reply docs/python/tutorial-flask.md]: reverted

File: docs/python/tutorial-flask.md (hunk 2 above)
Comment (gregvanl) [reply docs/python/tutorial-flask.md]: reverted

File: docs/python/tutorial-flask.md (hunk 3 above)
Comment (gregvanl) [reply docs/python/tutorial-flask.md]: Put back mention of Code on macOS

**Review by gregvanl — APPROVED** [review]
Thanks!

File: docs/python/tutorial-flask.md [hunk 4]
```diff
@@ -27,62 +27,30 @@ To successfully complete this Flask tutorial, you must do the following (which a
 1. Install the [Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python).
 
 1. Install a version of Python 3 (for which this tutorial is written). Options include:
-   - (All operating systems) A download from [python.org](https://www.python.org/downloads/); typically use the **Download Python 3.9.1** button that appears first on the page (or whatever is the latest version).
+   - (All operating systems) A download from [python.org](https://www.python.org/downloads/); typically use the **Download Python 3.11.1** button that appears first on the page (or whatever is the latest version).
```
Comment (brettcannon) [inline docs/python/tutorial-flask.md:30]:
```suggestion
   - (All operating systems) A download from [python.org](https://www.python.org/downloads/); typically use the **Download** button that appears first on the page.
```

**Review by brettcannon — APPROVED** [review]

**Pull request merged**